Assets/_BForBoss/_Entities/Scripts/LifeCycleManager.cs

```diff
@@ -0,0 +1,26 @@
+using UnityEngine;
+using System.Linq;
+
+namespace Perigon.Entities
+{
+    public class LifeCycleManager : MonoBehaviour
+    {
+        private LifeCycleBehaviour[] _lifeCycleBehaviours = null;
+
+        public int LivingEntities => _lifeCycleBehaviours.Count(life => life.IsAlive);
+
+        public void Initialize()
+        {
+            _lifeCycleBehaviours = FindObjectsOfType<LifeCycleBehaviour>();
```

Comment:
you can probably put this into Awake instead since `FindObjectsOfType` is a Unity dependent action. So you don't have to way for the Chain of `Initialize` from the top to call this.